collections/src/main/scala/strawman/collection/immutable/WithDefaultOps.scala

```diff
@@ -0,0 +1,13 @@
+package strawman.collection.immutable
+
+import strawman.collection.{IterableFactoryLike, MapFactory}
+import scala.{Option}
+
+private[immutable] trait WithDefaultOps[K, +V, +C <: Map[K, V]] { self: C =>
+
+  val underlying: C
+
+  val defaultValue: K => V
+
+  override def default(key: K): V = defaultValue(key)
+}
```

Comment:
This trait is so small, it seems that this abstract is not worth (I know it’s my fault that you tried to abstract over `Map.WithDefault` and `SortedMap.WithDefault` but now I see that this doesn’t seem to be useful).